class/defaults.yml

```
@@ -1,16 +1,17 @@
 parameters:
+  multi_instance: true
   keycloak:
     namespace: syn-keycloak
-    release_name: keycloak
+    release_name: keycloak-${_instance}
```

This could lead to odd internal naming if a user decides to instantiate the component with

```
applications:
  - keycloak as keycloak-int
  - keycloak as keycloak-prod
```

which seems natural, since the instance name will be used as the key for the per-instance settings in `parameters`.

I'd suggest simply using the instance name here:

```suggestion
    release_name: ${_instance}
```

Note: this configuration has the additional upside that the release name doesn't change for the unaliased component.